test/generators/avo_install_generator_test.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string-literal: true
+
+require 'test_helper'
+require 'generators/boring/avo/install/install_generator'
+
+class AvoInstallGeneratorTest < Rails::Generators::TestCase
+  tests Boring::Avo::InstallGenerator
+  setup :build_app
+  teardown :teardown_app
+
+  include GeneratorHelper
+
+  def destination_root
+    app_path
+  end
+
+  def test_should_install_avo_gem_successfully
```

Comment:
Let's update this to "test_should_configure_avo_gem" and merge tests from "test_should_configure_avo_gem_successfully" here as well since we don't have any options to check for OR edge cases to look out for this to be multiple test_ blocks.